test/case/infix_interfaces/bridge_fwd_sgl_dut.py

```diff
@@ -0,0 +1,86 @@
+#  ,---------------------------------------,       
+#  |                                       | 
+#  |                        br0            |  
+#  |                       /   \           | 
+#  | tgt:mgmt     tgt:data0     tgt:data1  | 
+#  '---------------------------------------'
+#        |                |     |            
+#        |                |     |           
+# ,-----------------------------------------,
+# |  host:mgmt   host:data0     host:data1  |
+# |              [10.0.0.1]     [10.0.0.2]  |
+# |                 (ns0)         (ns1)     |
+# |                                         |
+# |                 [ HOST ]                |
+# '-----------------------------------------'
+
+import infamy
+
+with infamy.Test() as test:
+    with test.step("Initialize"):
+        env = infamy.Env(infamy.std_topology("1x3"))
+        target = env.attach("target", "mgmt")
+
+    with test.step("Configure a bridge with dual physical port"):
+        _, tport0 = env.ltop.xlate("target", "data0")
+        _, tport1 = env.ltop.xlate("target", "data1")
+
+        target.put_config_dict("ietf-interfaces", {
+            "interfaces": {
+                "interface": [
+                    {
+                        "name": "br0",
+                        "type": "iana-if-type:bridge",
+                        "enabled": True,
+                    },
+                    {
+                        "name": tport0,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "bridge": "br0"
+                        }
+                    },
+                    {
+                        "name": tport1,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "bridge": "br0"
+                        }
+                    }
+                ]
+            }
+        })
+
+    with test.step("Ping host:data1 with IP 10.0.0.2 from host:data0 with IP 10.0.0.1"):
+        _, hport0 = env.ltop.xlate("host", "data0")
+        _, hport1 = env.ltop.xlate("host", "data1")
+
+        with infamy.IsolatedMacVlan(hport0) as ns0, infamy.IsolatedMacVlan(hport1) as ns1 :
+            port_conf_res = ns1.runsh("""
+            set -ex
+
+            ip link set iface up
+            ip addr add 10.0.0.2/24 dev iface
+            """)
+
+            if port_conf_res.returncode:
+                print(port_conf_res.stdout)
+                test.fail()
+
+            test_res = ns0.runsh("""
+            set -ex
+            
+            ip link set iface up
+            ip addr add 10.0.0.1/24 dev iface
+                                 
+            ping -c 5 -w 10 10.0.0.2 
+            """)
+
+            if test_res.returncode:
+                print(test_res.stdout)
+                test.fail()
```

Comment:
This section should probably use the new `addip` and `ping` helpers like the other tests.

Reply:
definitely :)